packages/phrases-ui/src/types.ts

```diff
@@ -1,19 +1,3 @@
-import { LanguageKey, languageKeyGuard } from '@logto/core-kit';
-
 import en from './locales/en';
 
 export type LocalePhrase = typeof en;
```

Comment:
maybe also move to index.ts?

Reply:
This type is used in locale files, we cannot move it to the index file.